src/commands/buildCommands/configs/rules.js

```diff
@@ -15,8 +15,8 @@ const postCssLoaderConfig = require('./postCssLoaderConfig.js');
 module.exports = (babelConfig, buildType) => {
 	const browserJSRules = {
 		// standard ES5 transpile through Babel
-		test: /\.jsx?$/,
-		include: [paths.src.browser.app, paths.packages.webComponents.src],
+		test: /\.(ts|js)x?$/,
+		include: [paths.src.browser.app, paths.packages.webComponents.src, paths.localPackages],
```

Comment:
I'm just learning how all of this gets wired up, but I'm not sure that you need to add `localPackages` as an `include`, because all the local package imports will get pulled in from `node_modules` (e.g. we never have an import path like `from '../../packages/my-package/thing.tsx'`. Have you confirmed that TS will _not_ work without this change?

Reply:
I confirm that source maps don't work without adding ts/tsx to babel-loader config. I've tried to add them from `node_modules` like `paths.packages.webComponents.src`

Reply:
One more note - we don't need this rule in serverJSRules, only in browserJSRules, so I'll remove it

Reply:
interesting, thanks for looking into it. We'll probably be reviewing some of this config as we start focusing on build optimizations a bit more, so this is good context.